WORKSPACE

```diff
@@ -106,3 +106,13 @@ new_http_archive(
     type = "zip",
     url = "https://github.com/nunit/nunitv2/releases/download/2.6.4/NUnit-2.6.4.zip",
 )
+
+maven_jar(
+    name = "scala-xml",
+    artifact = "org.scala-lang.modules:scala-xml_2.11:1.0.5",
+)
+
+maven_jar(
+    name = "scalatest",
+    artifact = "org.scalatest:scalatest_2.11:2.2.4",
```

Comment:
Yup, I figured if we need some of the optionals in there, we can still add them manually when the need comes up.